### ISSUE TITLE
bug: Element has shown in dom but not display in page

### ISSUE THREAD

**Issue by xyxabcdiy** [driftyco/ionic#4809]
**Type**: <span ionic-type>bug</span>

**Platform**: <span ionic-platform>android</span> <span ionic-platform-version>5.x</span>  <span ionic-webview>webview</span>

<span ionic-description>I use camera plugin to choice image in the local gallery, and display image in `img` tag. But sometimes, the element has show in the dom and image resourse has finished install, but it not display in the page.

`HTML` code:
```
<img ng-repeat="image in imageList" ng-src="{{image}}">
```
`JavaScript` code:
```
var options = {
                destinationType: navigator.camera.DestinationType.FILE_URI,
                sourceType: navigator.camera.PictureSourceType.PHOTOLIBRARY,
                saveToPhotoAlbum: true,
                targetWidth: 800,
                targetHeight: 800
            };
navigator.camera.getPicture(function (imageURI) {
          $scope.imageList.push(imageURI);
                if (!$scope.$$phase) {
                    $scope.$apply();
                }
            }, function (error) {
                console.log("Error: " + error);
            }, options);
```

<span is-issue-template></span>

**Comment by Ionitron** [comment]
Greetings @xyxabcdiy!

My sensors indicate that you need to update your issue through our [custom issue form](http://ionicframework.com/submit-issue/#/?iid=4809). We are now requiring all issues to be submitted this way, to ensure that we have all of the information necessary to fix them as quickly as possible.

**[Click Here To Update Your Issue](http://ionicframework.com/submit-issue/#/?iid=4809)**

I will have no choice but to close this issue if it is not resubmitted through the form. Please fill out the rest of the form, so that I may use my friendly robot powers to assist you.

Thank you!

<span ionitron-issue-resubmit></span>

**Comment by xyxabcdiy** [comment]
supplement: Physical back button does not work and if click other input element, image will display soon and everything goes well. But naxt choice local image will cause the problem again.

**Comment by PerfectionCSGO** [comment]
Are you witelisting file:// or blob:// ?

**Comment by PerfectionCSGO** [comment]
For example, try to include below into your project.

    angular.module('starter').config(function( $compileProvider) {
        $compileProvider.imgSrcSanitizationWhitelist(/^\s*(https?|ftp|file|ms-appx|cdvfile|ms-appdata|blob|x-wmapp0):|data:image\//);
        $compileProvider.aHrefSanitizationWhitelist(/^\s*(https?|ftp|mailto|file|cdvfile|ghttps?|ms-appx|x-wmapp0):/);
    });

**Comment by xyxabcdiy** [comment]
That's for sure. Some times i can display local image in the page. If image can't display, it's innerHtml will be :
```
<img ng-src="file:///storage/emulated/0/Android/data/com.ionicframework.app755472/cache/modified.jpg?1450361509542" image-load="onImageLoad($event)"  data-ng-animate="2" src="file:///storage/emulated/0/Android/data/com.ionicframework.app755472/cache/modified.jpg?1450361509542" class>
```
I don't known the `data-ng-animate='2'` in the element attribute and if it can be show, it will not have this attribute.

**Comment by d1sa** [comment]
at the same time, the top bar also displays or hides?

**Comment by xyxabcdiy** [comment]
@d1sa Top bar displays and the bugs has fixed in 1.2.1

**Comment by xyxabcdiy** [comment]
The bugs has fixed in 1.2.1
